typescript/provider-sdk/src/ism.ts

```diff
@@ -42,3 +42,21 @@ export type IsmModuleAddresses = {
   deployedIsm: string;
   mailbox: string;
 };
+
+export interface RawDomainRoutingIsmConfig {
+  type: 'domainRoutingIsm';
+  owner: string;
+  domains: Record<string, string>;
+}
+
+export interface RawIsmConfigs {
+  domainRoutingIsm: RawDomainRoutingIsmConfig;
+  merkleRootMultisigIsm: MultisigIsmConfig;
+  messageIdMultisigIsm: MultisigIsmConfig;
+  testIsm: TestIsmConfig;
+}
+
+export interface IsmArtifact<T extends keyof RawIsmConfigs> {
+  config: RawIsmConfigs[T];
+  addresses: { deployedIsm: string };
```

Comment:
no more `mailboxAddress` 🔥

Reply:
I didn't see the need for Radix ISM operations. Was it added for Aleo?

Reply:
instead of having the mailbox address between the addresses, making it part of the ism config should cover those cases i think